File: packages/internal/generators-files/quarkus/library/shared/java/src/test/java/__packageDirectory__/GreetingServiceTest.java
package <%= packageName %>;

import jakarta.inject.Inject;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

import io.quarkus.test.junit.QuarkusTest;

@QuarkusTest
public class GreetingServiceTest {

    @Inject 
    GreetingService service;

    @Test
    public void testGreetingService() {
        Assertions.assertEquals("Hello World!", service.greeting());
    }
}

File: packages/internal/generators-files/quarkus/library/files/kotlin/src/test/kotlin/__packageDirectory__/GreetingServiceTest.kt
package <%= packageName %>

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test
import io.quarkus.test.junit.QuarkusTest
import jakarta.inject.Inject


@QuarkusTest
class GreetingServiceTest {


  @Inject
  lateinit var greetingService: GreetingService


  @Test
  fun `Should return Hello World`() {
    println(">> Should return Hello World")
    Assertions.assertTrue(greetingService.greeting().contains("Hello World"));
  }


}

